packages/radix-ui-themes/src/components/card.tsx

```diff
@@ -43,7 +43,7 @@ const Card = React.forwardRef<CardElement, CardProps>((props, forwardedRef) => {
         withMarginProps(marginProps)
       )}
     >
-      {asChild ? getChild() : children}
+      {asChild ? getChild() : <div className="rt-CardInner">{children}</div>}
```

Comment:
Like this. Should be equivalent, unless I'm missing something?

```suggestion
       <div className="rt-CardInner">
        {asChild ? getChild() : children}
      </div>
```

Reply:
That's the code we had before removing `CardInner`—I actually had tried the same as you suggested and it hadn't worked with `asChild`. I didn't investigate and assumed there was a reason you had done it this way.

Reply:
haha ok, there must be then 😀 I remember it has to do with when using `<Card asChild><a>`. Let's keep as is then.